frontend/src/containers/ThankYouPage/ThankYouPage.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import './ThankYouPage.css';
+// import WelcomePage from '/containers/WelcomePage/WelcomePage';
+import { Button } from '@mui/material';
+import { amber } from '@mui/material/colors';
+
+const ThankYouPage: React.FC = () => {
+    return (
+      <>
+        <div className='content__thank-you'>
+            <div className='container'>
+                <h2>Thank you for your Feedback!</h2>
+                <h3>We're grateful for your feedback</h3>
+                <p>We will keep on improving our services and feature.</p>
+                <Button 
+                    variant="contained"
+                    sx={{ 
+                        bgcolor: amber[900],
```

Comment:
Please update this to "#f36b3b". Let's update this once Gren's PR has been merged.